### PR TITLE
fix #7: await the async wrapped function

### LINKED ISSUE #7: TypeError when clicking L.Rest then Rest [Bug]
The "Select Dice to Roll" appears, and also appears to function.  But now the "Rest" Button no longer works, on the Long Rest screen.

```
lr-hd-healing.js:360 Uncaught (in promise) TypeError: results.push is not a function
[Detected 3 packages: long-rest-hd-healing, lib-wrapper, system:dnd5e]
    at _recoverUses (lr-hd-healing.js:360:17)
    at _recoverItemUses (lr-hd-healing.js:347:17)
    at Actor5e.patched_getRestItemUsesRecovery (lr-hd-healing.js:324:17)
    at 🎁call_wrapper [as call_wrapper] (libWrapper-wrapper.js:616:16)
    at 🎁CONFIG.Actor.documentClass.prototype._getRestItemUsesRecovery#0 (libWrapper-wrapper.js:189:20)
    at Actor5e._rest (actor.mjs:1494:23)
    at Actor5e.patchedLongRest (lr-hd-healing.js:195:25)
````

image

image

Only modules enabled are "Find the culprit", "libWrapper" (used by many modules) and "Long Rest Hit Die Healing for D&D5E".

Foundry v10.288.  Long Rest module 3.3.0.

### PR DESCRIPTION
**Pull request by kaelad02**

### CODE REVIEW

**Comment by mhilbrunner** [comment]
I tested this and can confirm it fixes the issue in long resting for actors containing rechargeable items (https://github.com/a-ws-m/FVTT-Long-Rest-HD-Healing-5e/issues/7).

I created a temporary fork and release with this fix applied as I need it for tomorrow's session, if anyone can't wait until this is merged:
`https://raw.githubusercontent.com/mhilbrunner/FVTT-Long-Rest-HD-Healing-5e/master/module.json`

It may go away as soon as this is merged, so I recommend checking and switching back to the official repository then :)

**Comment by kiloforce** [comment]
Please make this so

**Review by a-ws-m — APPROVED** [review]

**Pull request merged**
